diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -208,6 +208,7 @@
 			elements: await Promise.all(tags.params.map(onParam))
 		});
 		if (entry.description) node.addJsxProperty("description", await renderer.renderMarkdownToHast(entry.description));
+		if (entry.deprecated) node.addExpressionNode("deprecated", valueToEstree(true));
 		return node.build();
 	}
 	async function onParam(param) {
